.golangci.yml

```diff
@@ -268,25 +268,25 @@ issues:
   - linters:
     - revive
     text: "package-comments"
-    path: ^(main|(packaging|apis|controllers|feature|test|pkg)\/.*)\.go$
+    path: ^(apis/(v1alpha3|v1alpha4)\/.*)\.go$
```

Comment:
I left the ignore on these packages - they're going to be removed relatively soon so I think it's find to leave them as is instead of fixing them up.